blues/ncmc.py

```diff
@@ -263,6 +263,9 @@ def calculate_com(self,  pos_state, total_mass=None, mass_list=None, residueList
         lig_coord = lig_coord*unit.nanometers
         copy_coord = copy.deepcopy(lig_coord)
         #mass corrected coordinates (to find COM)
+        print('mass_list', mass_list)
```

Comment:
Why are we printing these? Is this for debug purposes? If so, can you just make an optional `debug` argument and have them printed only if debug is True? Presumably you don't want a log file somewhere filling up with tens of thousands of debug statements.